doxia-modules/doxia-module-markdown/pom.xml

```diff
@@ -72,5 +72,80 @@ under the License.
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+    </dependency>
   </dependencies>
+  <build>
+
+    <plugins>
+
+      <!-- install -->
+      <plugin>
+        <artifactId>maven-install-plugin</artifactId>
```

Comment:
Aren't they available when `verify` run because this is run after `install`?

Reply:
I wanted to make sure `mvn verify` runs successfully on the reactor, without the need to run `mvn install` before.

Also, the artifacts produced by other modules are not retrieved by **maven-invoker-plugin** (even with its `<extraArtifacts>` configuration property). That would be a nice enhancement on this plugin! ;-)

Reply:
hm... I need to think about this. It is somewhat hacky, but I do understand your intentions.

Reply:
It is hacky, I agree, especially as it specifies other artifacts by their path relative to this module. At the same time, it's really the purpose of **maven-install-plugin**: install artifacts in a Maven repository, which is what we're doing here so that **maven-invoker-plugin** has all it needs to run integration tests.

I'm open to alternatives (as that would be useful for other multi-module projects!).

Reply:
@rfscholte @khmarbaise Do you have an opinion here?